src/components/image/image.tsx

```diff
@@ -0,0 +1,110 @@
+import { Component, Element, h, Prop } from '@stencil/core';
+
+/**
+ * Image is a component that displays an image.
+ * It supports lazy loading of images to improve performance,
+ * the image will only be loaded when it enters the viewport.
+ *
+ * @private
+ *
+ * @exampleComponent limel-example-image-basic
+ * @exampleComponent limel-example-image-lazy
+ * @exampleComponent limel-example-image-interactive
+ * @exampleComponent limel-example-image-gallery
+ * @exampleComponent limel-example-image-no-source
+ */
+@Component({
+    tag: 'limel-image',
+    shadow: false,
+    styleUrl: 'image.scss',
+})
+export class ImageComponent {
+    /**
+     * The image source URL
+     */
+    @Prop()
+    public src: string;
+
+    /**
+     * Alternative text for the image
+     */
+    @Prop()
+    public alt: string = '';
+
+    /**
+     * Enable lazy loading for images
+     */
+    @Prop()
+    public lazyLoadImages = false;
```

Comment:
```suggestion
```suggestion
    /**
     * Enable lazy loading of the image
     */
    @Prop()
    public lazyLoad = false;
```
```